src/app/feature/certificate/pages/certificate/certificate.component.scss

```
@@ -0,0 +1,522 @@
+	letter-spacing: 3px;
+	margin: 0;
+}
+
+.certificate--minimalist .certificate__body {
+	flex: 1;
+}
+
+.certificate--minimalist .certificate__recipient-simple {
+	font-size: 48px;
+	color: #000;
+	font-family: 'Roboto', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)